tethys_dataset_services/engines/geoserver_engine.py

```diff
@@ -2270,40 +2489,106 @@ def delete_store(self, store_id, purge=False, recurse=False, debug=False):
             self._handle_debug(response_dict, debug)
             return response_dict
 
-    def delete_style(self, style_id, purge=False, recurse=False, debug=False):
+    def delete_coverage_store(self, store_id, recurse=True, purge=True):
         """
-        Delete a style.
+        Delete the specified coverage store.
 
         Args:
-          style_id (string): Identifier of the style to delete.
-          purge (bool, optional): Purge if True.
-          recurse (bool, optional): Delete recursively if True.
-          debug (bool, optional): Pretty print the response dictionary to the console for debugging. Defaults to False.
+            store_id (string): Identifier for the store to be deleted. Can be a store name or a workspace name combination (e.g.: "name" or "workspace:name"). Note that the workspace must be an existing workspace. If no workspace is given, the default workspace will be assigned.
+            workspace (str): the name of the workspace to add the style to
+            name(str):name of the coverage
```

Comment:
Remove these doc strings. Replaced by store_id (typo in the workspace doc string btw)